### ISSUE TITLE
Build fails if -DWITH_SDL2 specified

### ISSUE THREAD

**Issue by Fat-Zer** [instead-hub/instead#91]
Error message:

```
FAILED: src/CMakeFiles/sdl-instead.dir/SDL_gif.c.o 
/usr/bin/x86_64-pc-linux-gnu-gcc -DGAMES_PATH=\"/usr/share/instead/games/\" -DICON_PATH=\"/usr/share/pixmaps/\" -DLANG_PATH=\"/usr/share/instead/lang/\" -DSTEAD_PATH=\"/usr/share/instead/stead/\" -DTHEMES_PATH=\"/usr/share/instead/themes/\" -DVERSION=\"3.3.4\" -D_HAVE_ICONV -D_USE_BROWSE -D_USE_GTK -D_USE_SDL -D_USE_UNPACK -Dunix -I/usr/include/lua5.1 -I/usr/include/SDL2 -I/usr/include/gtk-3.0 -I/usr/include/pango-1.0 -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/lib64/libffi/include -I/usr/include/harfbuzz -I/usr/include/freetype2 -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/fribidi -I/usr/include/uuid -I/usr/include/cairo -I/usr/include/pixman-1 -I/usr/include/libpng16 -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/gio-unix-2.0 -I/usr/include/atk-1.0 -I/usr/include/at-spi2-atk/2.0 -I/usr/include/dbus-1.0 -I/usr/lib64/dbus-1.0/include -I/usr/include/at-spi-2.0  -O2 -pipe -march=x86-64 -frecord-gcc-switches   -Wall -Wextra -Wno-unused-parameter -MD -MT src/CMakeFiles/sdl-instead.dir/SDL_gif.c.o -MF src/CMakeFiles/sdl-instead.dir/SDL_gif.c.o.d -o src/CMakeFiles/sdl-instead.dir/SDL_gif.c.o -c /var/tmp/portage/games-engines/instead-3.3.4/work/instead-3.3.4/src/SDL_gif.c
/var/tmp/portage/games-engines/instead-3.3.4/work/instead-3.3.4/src/SDL_gif.c:744:21: error: conflicting types for ‘IMG_LoadGIFAnimation_RW’; have ‘Animation_t *(SDL_RWops *)’
  744 | static Animation_t *IMG_LoadGIFAnimation_RW(SDL_RWops *src)
      |                     ^~~~~~~~~~~~~~~~~~~~~~~
In file included from /var/tmp/portage/games-engines/instead-3.3.4/work/instead-3.3.4/src/SDL_gif.c:26:
/usr/include/SDL2/SDL_image.h:168:41: note: previous declaration of ‘IMG_LoadGIFAnimation_RW’ with type ‘IMG_Animation *(SDL_RWops *)’
  168 | extern DECLSPEC IMG_Animation * SDLCALL IMG_LoadGIFAnimation_RW(SDL_RWops *src);
      |      
      |                                         ^~~~~~~~~~~~~~~~~~~~~~~
```

The complete build log can be found in Gentoo bug: https://bugs.gentoo.org/796962. It refers to version 3.3.4, though the same issue is applicable to the master.

Issue can be easily solved by renaming `IMG_LoadGIFAnimation_RW`. Though, I suppose you may want to use `IMG_LoadGIFAnimation_RW` from the library rather than the bundled version.

**Comment by gl00my** [comment]
Thank you! IMG_LoadGIFAnimation_RW is abesent in older versions of SDL_image, so i just renamed it.
Write me if this solved  the problem.

**Comment by Fat-Zer** [comment]
Yep, it did. Thanks for instant reaction.

**Comment by gl00my** [comment]
I have released bugfix version 3.3.5. You can use it as an upstream version.
